tests/keystore_tests.rs

```diff
@@ -13,8 +12,8 @@ use crate::common::{create_tmp_config, daemon, CommonArgs};
 
 // https://github.com/ChainSafe/forest/issues/2499
 #[test]
-fn forest_headless_encrypt_keystore_no_passphrase_should_fail() -> Result<()> {
-    let (config_file, _data_dir) = create_tmp_config()?;
+fn forest_headless_encrypt_keystore_no_passphrase_should_fail() -> anyhow::Result<()> {
```

Comment:
Did you overlook these tests or do you really want them to return a `Result`?

```suggestion
fn forest_headless_encrypt_keystore_no_passphrase_should_fail() {
```

Reply:
Finished an updated the description

Reply:
I'm not sure what happened here